gulp/tasks/clean.js

```diff
@@ -1,21 +1,21 @@
+/* eslint-env node */
```

Comment:
I thought I could just do 

`./gulp/.eslintrc.json`
```
{
  "env": {
    "node: true
  }
}
```

but that wasn't working for some reason. I'll take another look at it before I merge this. Got sidetracked when my jenkins build finished :)